### PR TITLE
Add dark mode to login/join/waitlist screens (see designs - based on system preference)

### PR DESCRIPTION
**Pull request by Excellent-coding**
https://linear.app/find-labs/issue/ENG-32/add-dark-mode-to-loginjoinwaitlist-screens-see-designs-based-on-system

### CODE REVIEW

File: apps/web-app/src/components/ThemeSelectButton.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { MoonIcon, SunIcon } from "@radix-ui/react-icons";
```
Comment (khalil-omer) [inline apps/web-app/src/components/ThemeSelectButton.tsx:1]: This component is not in the designs. We might use it in the future, but it should not be added to the auth pages.

File: apps/web-app/src/components/Navbar.tsx [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
       <Link href={"/"}>
         <div className="flex cursor-pointer">
-          <img src="/logo.svg" alt="logo" />
-          <div className="text-bold text-xl my-auto ml-3">find</div>
+          <Image src="/logo.svg" width={32} height={32} alt="logo" />
+          <div className="text-bold text-xl my-auto ml-3 dark:text-gray-100">find</div>
         </div>
       </Link>
       <div className="flex">
+        <ThemeSelectButton />
```
Comment (khalil-omer) [inline apps/web-app/src/components/Navbar.tsx:36]: You can create this component for the future, but it is not part of the designs right now so we should not add the ThemeSelectButton to any pages.

**Review by khalil-omer — CHANGES_REQUESTED** [review]
This is supposed to be based on system setting,

File: apps/web-app/src/components/ThemeSelectButton.tsx (hunk 1 above)
Comment (Excellent-coding) [reply apps/web-app/src/components/ThemeSelectButton.tsx:1]: got it

**Review by khalil-omer — APPROVED** [review]

**Pull request merged**